sonarqube/content.md

```diff
@@ -92,6 +92,16 @@ $ docker build --tag=sonarqube-custom .
 $ docker run -ti sonarqube-custom
 ```
 
+### Avoid hard termination of SonarQube
+
+Starting from SonarQube 7.8, SonarQube stops gracefully, waiting for any tasks in progress to finish.
+Waiting for in-progress tasks to finish can take a large amount of time which the docker does not expect by default when stopping.
+To avoid having the SonarQube instance killed by the Docker daemon after 10 seconds, it is best to configure a timeout to stop the container with `--stop-timeout`. For example:
```

Comment:
The travis test that runs the `markdownfmt` check needs these three lines to be combined to a single line. The other travis error about a changed `README.md` is false and so no action need to be taken for that.

```diff
diff ./sonarqube/content.md markdownfmt/./sonarqube/content.md
--- /tmp/markdownfmt986287976
+++ /tmp/markdownfmt671901095
@@ -94,9 +94,7 @@
 
 ### Avoid hard termination of SonarQube
 
-Starting from SonarQube 7.8, SonarQube stops gracefully, waiting for any tasks in progress to finish.
-Waiting for in-progress tasks to finish can take a large amount of time which the docker does not expect by default when stopping.
-To avoid having the SonarQube instance killed by the Docker daemon after 10 seconds, it is best to configure a timeout to stop the container with `--stop-timeout`. For example:
+Starting from SonarQube 7.8, SonarQube stops gracefully, waiting for any tasks in progress to finish. Waiting for in-progress tasks to finish can take a large amount of time which the docker does not expect by default when stopping. To avoid having the SonarQube instance killed by the Docker daemon after 10 seconds, it is best to configure a timeout to stop the container with `--stop-timeout`. For example:
```

Reply:
Alright, fixed.